### DISCUSSION TITLE
Any way to get balance?

### DISCUSSION THREAD

**Discussion by DiMiTriFrog** [ejtraderLabs/ejtraderCT#22] [General]
is there any way to get balance of account from lib?

**Comment by traderpedroso** [comment]
> is there any way to get balance of account from lib?

Can you please clarify your question and specify which library you're referring to? The context here is a bit unclear, and it's hard for me to provide an accurate response without further information.

Regardless, all financial market libraries have a balance check option. Please refer to the readme for further details.

**Reply by DiMiTriFrog** [reply]
Ejctrader.
I cant get the balance

**Reply by traderpedroso** [reply]
no account information is support by ctrader fix its not library limitation

**Reply by DiMiTriFrog** [reply]
Oh… fuck.
Thanks!

**Reply by traderpedroso** [reply]
It's simple  I manage it in the following way: I add the initial balance to the open positions, whether they are market or pending. You can check the gain on the position and I simply calculate the gain or loss based on the initial balance.
